ark/spLDA/processing.py

```diff
@@ -0,0 +1,86 @@
+import functools
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import os
+import pickle
+from ark.utils.spatial_lda_utils import check_format_cell_table_args
+# from spatial_lda.featurization import *
+# from spatial_lda.visualization import plot_adjacency_graph()
+
+
+def format_cell_table(cell_table, markers=None, clusters=None, fovs="all_fovs"):
+    """
+    Formats a cell table containing one for more fields of view to be compatible with the spatial_lda library.
+    Args:
+        cell_table (pd.DataFrame):
+            A pandas DataFrame containing the columns of cell marker frequencies and/or cluster ids.
+        markers (list, str):
+            A list of strings corresponding to marker names.
+            Default: None
+        clusters (list, int):
```

Comment:
Should specify in clusters and markers docstring something to the effect of "either clusters or markers must be provided"

Reply:
Can `clusters` be a single `int` or is it just a list of `int`. If the latter, you do not need to specify `list, int`, just `list`. The user can figure out from the documentation that the `list` should always contain `int`.